src/nbl/video/CVulkanPhysicalDevice.cpp

```diff
@@ -1196,6 +1199,9 @@ std::unique_ptr<CVulkanPhysicalDevice> CVulkanPhysicalDevice::create(core::smart
             properties.limits.workgroupMemoryExplicitLayout16BitAccess = workgroupMemoryExplicitLayout.workgroupMemoryExplicitLayout16BitAccess;
         }
 
+        if (isExtensionSupported(VK_KHR_RAY_TRACING_POSITION_FETCH_EXTENSION_NAME) && isExtensionSupported(VK_KHR_ACCELERATION_STRUCTURE_EXTENSION_NAME))
+            properties.limits.rayTracingPositionFetch = rayTracingPositionFetchFeatures.rayTracingPositionFetch;
+
         if (isExtensionSupported(VK_EXT_COLOR_WRITE_ENABLE_EXTENSION_NAME))
             properties.limits.colorWriteEnable = colorWriteEnableFeatures.colorWriteEnable;
 #if 0 //TODO
```

Comment:
somewhere in createLogicalDevice_impl you need `REQUIRE_EXTENSION_IF(enabledFeatures.accelerationStructure && limits.rayTracingPositionFetch,...)` then an `enabledFeatures.rayTracingPositionFetch = ` based on the same condition

Reply:
Hmm, how do you do `enabledFeatures.rayTracingPositionFetch = ` if it's no longer a field in `SPhysicalDeviceFeatures`? I thought it was `MOVE_TO_LIMIT`

Reply:
mistyped, the thing you're supposed to set is the `VkRayTracingPositionFetchFeatures.rayTracingPositionFetch` struct